api/tests/routes/backoffice/users_test.py

```diff
@@ -105,6 +106,58 @@ def test_suspend_beneficiary_user(self, client, beneficiary_fraud_admin):
         assert user.action_history[0].extraData["reason"] == users_constants.SuspensionReason.FRAUD_RESELL_PRODUCT.value
         assert user.action_history[0].comment == "Le jeune a mis en ligne des annonces de vente le lendemain"
 
+    @pytest.mark.parametrize(
+        "suspen_data",
```

Comment:
Plutôt que décomposer le tuple ensuite, tu peux écrire directement ici : 
```suggestion
        "reason,cancel_bookings,cancel_event",
```

Reply:
je ne connaissait pas cet usage de parametrize, merci :+1: